src/main/java/com/ericsson/ei/mongo/MongoDBHandler.java

```diff
@@ -124,7 +125,8 @@ public void insertDocument(String dataBaseName, String collectionName, String in
      * @param document - Document object to insert
      * @throws MongoWriteException
      */
-    public void insertDocumentObject(String dataBaseName, String collectionName, Document document) throws MongoWriteException {
+    public void insertDocumentObject(String dataBaseName, String collectionName, Document document, MongoCondition condition, String eventId) 
```

Comment:
add the method parameters in the java doc above

Reply:
updated method parameters in Javadoc